spec/models/item_spec.rb

```diff
@@ -1,5 +1,51 @@
-# require 'rails_helper'
-#
-# RSpec.describe Item, type: :model do
-#   pending "add some examples to (or delete) #{__FILE__}"
-# end
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.